troubleshooting.md

```diff
@@ -98,6 +98,44 @@ When doing a Podman command such as `build`, `commit`, `pull`, or `push` to a re
 TLS verification is turned on by default.  If encryption is not used with
 those commands, this error can occur.
 
+Or to avoid using the `--tls-verify=false` option or add insecure registry config.
```

Comment:
Added in the wrong place, this needs to be below the actual `tls-verify` documentation